Source/LinuxParking.API/Domain/Models/Customer.cs

```diff
@@ -0,0 +1,18 @@
+using System.ComponentModel.DataAnnotations;
```

Comment:
We already have the user table from the Identity service, it should be utilized instead.